playbooks/gluster-volume.yml

```diff
@@ -20,8 +20,8 @@
   tasks:
   - include_role:
       name: gluster-secure-shared-storage
```

Comment:
(attempting to) Explaining the commit message more clearly:
- Say there is a playbook that includes this role for only a certain subset of the hosts in the cluster group
- Further assume that all hosts that are part of this cluster group are part of the play

Then the play will not error out in the check to ensure all hosts are part of the play. This is because play_hosts has them, but play itself is included for a subset, and also because for the life of me I am not able to figure out how to get a list of hosts on which this play is included.

As long as we use these playbooks and ensure future playbooks do not break this, things work as required.